frontend/src/app/services/covid.service.ts

```diff
@@ -23,6 +24,10 @@ export class CovidService {
       .toPromise().then(item => (item as { situations: HospitalBedsDaily[] }).situations);
   }
 
+  public getGeneralSituationPerDate(): Promise<GeneralSituationDaily[]> {
+    return this.http.get(this.apiUrl + '/daily/tableData/10')
```

Comment:
/daily/tableData does not exist anymore

Reply:
dann sag ma bitte wie i den node_modules im gitignore eintragen soll...hab das eigentlich extra gmacht

Reply:
ausadem sind da zwar node module Folder ana im frontend und ana herausen, welcher ghert weg?

Reply:
Es gehören beide weg, du hast sie aber schon committed also musst sie iwie wieder aus da version control rauslöschen (da reicht jetz das gitignore nit) i weiß aber jetz nit wie genau da musst googlen.

Reply:
I glaub jetz sind sie weg